src/njoy21.hpp

```diff
@@ -18,7 +18,11 @@
 #include "Log.hpp"
 #include "utility.hpp"
 
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Might it be possible to tweak the offending lipservice code so that this isn't necessary? (I suppose I could go take a look myself.) Better a fundamental fix than diagnostic push/pop pragmas everywhere else that uses it.

Reply:
As you saw in #106, the lipservice code is valid and initialized. I do think think this is a GCC bug.